keps/sig-apps/3939-allow-replacement-when-fully-terminated/README.md

```diff
@@ -800,6 +888,9 @@ feature enablement causes the number of syncs to increase.
 
 #### Are there any missing metrics that would be useful to have to improve observability of this feature?
 
+We identified that a new metric would be necessary to improve observability.  
```

Comment:
From above:

> job_pods_creation_total (new) the action label will mention what triggers creation

I don't see that metric in k/k, was this added?

Reply:
Please fill in SLO question.

Reply:
> From above:
> 
> > job_pods_creation_total (new) the action label will mention what triggers creation
> 
> I don't see that metric in k/k, was this added?

I have a section below.  I was told that adding metrics in alpha is not allowed so I will be adding a new metric in beta.